dd-library-php-setup.php

```diff
@@ -884,6 +1163,87 @@ function get_ini_template($requestInitHookPath)
     // phpcs:enable Generic.Files.LineLength.TooLong
 }
 
+function get_ini_content_appsec($helperPath, $rulesPath)
+{
+    // phpcs:disable Generic.Files.LineLength.TooLong
+    return <<<EOD
+; Loads the dd-appsec extension
+extension = ddappsec.so
+
+; Enables or disables the loaded dd-appsec extension.
+; If disabled, the extension will do no work during the requests.
+; This value is ignored on the CLI SAPI, see ddappsec.enabled_on_cli.
+;ddappsec.enabled = Off
```

Comment:
For consistency with the tracer (`datadog.trace.*`) and profiling (`datadog.profiling.*`), can we move the naming here to `datadog.appsec.*`?